.github/ISSUE_TEMPLATE/design-intent-collaboration.md

```diff
@@ -48,7 +48,7 @@ Please note:
 - Please select a time slot at least 2 business days from request date.
 
 ## Additional information
-For the Design Intent, you'll receive feedback from design, accessibility, and IA only. You won't receive feedback from content and QA, as it's too early in the development process.
+For the Design Intent, you'll receive feedback from design and IA only. You won't receive feedback from content and QA, as it's too early in the development process.
```

Comment:
We say you won't receive feedback from content and QA because xyz... Can you add a note saying you won't receive feedback from accessibility be they are no longer part of this touchpoint due to bandwidth constraints?

Reply:
This has been updated